frontend/src/components/shared/ErrorPanel.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <transition name="slide-fade">
+    <article v-if="message" id="errorPanel" class="message is-danger">
+      <div class="message-header">
+        <b>{{ title }}</b>
+        <button v-if="hasHideListener" class="delete" aria-label="delete" @click="$emit('hideErrorPanel')" />
+      </div>
+      <div class="message-body has-text-centered">
+        <h5 class="title is-6">{{ message }}</h5>
+      </div>
+    </article>
+  </transition>
+</template>
+
+<script>
+
+/**
+ * The ErrorPanel component is controlled by providing a `message`, and
+ * an optional `title`.
+ *
+ * The `@hideErrorPanel` event could be used to reset
+ * the `message` by clicking the "x" button.
+ *
+ * Example usage for the ErrorPanel component:
+ * <ErrorPanel :message="mess" @hideErrorPanel="mess=''" />
+ */
+export default {
+  name: 'ErrorPanel',
+  props: {
+    title: {
+      type: String,
+      default: 'Oops!..',
+    },
+    message: {
+      type: String,
+    },
+  },
+  computed: {
+    hasHideListener() {
+      return Object.keys(this.$listeners).includes('hideErrorPanel');
+    },
+  },
+};
+
+</script>
+
+<style>
+
+#errorPanel {
+  z-index: 11;
+  position: fixed;
+  width: 350px;
+  right: 20px;
+  bottom: 80px;
```

Comment:
This error only occurs in critical cases, so it should be more visible to the user. Instead of the corner of the page, could we have in the middle at the bottom, or in the center of the page?

Reply:
I think at the bottom is more suitable if we want to keep this animation. I don't feel too strongly about it bottom-center vs bottom-right so I can move it to the center.

Reply:
Bottom-center would make it more visible, thanks.

Reply:
A commit has been added to address this. It also includes the change to always display the error panel even for 404 responses.

Reply:
Nice!